tests/api/v1/samples/upsertBulk.js

```diff
@@ -297,58 +297,6 @@ describe('api: POST ' + path, () => {
     });
   });
 
-  it('upsert with readOnly field status should fail', (done) => {
```

Comment:
maybe just ".skip" these now since we'll need these tests again once we do the story you just created?